chef_master/source/knife_supermarket.rst

```diff
@@ -1,17 +1,9 @@
 =====================================================
 knife supermarket
 =====================================================
-`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/plugin_knife_supermarket.rst>`__
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/knife_supermarket.rst>`__
 
-.. warning:: Only use knife supermarket if you are using a Chef 12.12 or earlier. If you are using Chef 12.13 or later, you should use the `knife cookbook site </knife_cookbook_site.html>`__ commands.
-
-.. tag supermarket_api_summary
-
-The Supermarket API is used to provide access to cookbooks, tools, and users on the `Chef Supermarket <https://supermarket.chef.io>`__. All of the cookbooks, tools, and users on the Supermarket are accessible through a RESTful API by accessing ``supermarket.chef.io/api/v1/`` via the supported endpoints. In most cases, knife is the best way to interact with the Supermarket; however in some cases, direct use of the Supermarket API is necessary.
-
-.. end_tag
-
-The ``knife supermarket`` subcommand is used to interact with cookbooks that are located in private Chef Supermarket configured inside the firewall. A user account is required for any community actions that write data to the Chef Supermarket; however, the following arguments do not require a user account: ``download``, ``search``, ``install``, and ``list``.
+The ``knife supermarket`` subcommand is used to interact with cookbooks that are located in on the public Supermarket as well as private Chef Supermarket sites. A user account is required for any community actions that write data to the Chef Supermarket; however, the following arguments do not require a user account: ``download``, ``search``, ``install``, and ``list``.
```

Comment:
All kinds of passive constructions:
Use the ``knife supermarket`` subcommand  to interact with cookbooks located in both the public Supermarket and private Chef Supermarket sites. For actions that write data to the Chef Supermarket, you'll need a user account. You do not need a user account to use ``download``, ``search``, ``install``, and ``list`` for the Chef Supermarket because these are not ``write`` actions.

Reply:
This is actually the standard text we use for all the subcommands. It probably needs to get changed, but you'll find a nearly identical paragraph is about 20 places. We should fix those all at once if we're going to do it.

Reply:
I'm fixing this site wide in my knife.rb branch.